aviary/subsystems/aerodynamics/flops_based/test/test_solved_aero_group.py

```diff
@@ -91,6 +88,80 @@ def test_solved_aero_pass_polar(self):
 
         prob = AviaryProblem()
 
+        prob.load_inputs(
+            "subsystems/aerodynamics/flops_based/test/data/high_wing_single_aisle.csv", ph_in)
+
+        prob.aviary_inputs.set_val(Aircraft.Design.LIFT_POLAR,
+                                   np.zeros_like(CL), units='unitless')
+        prob.aviary_inputs.set_val(Aircraft.Design.DRAG_POLAR,
+                                   np.zeros_like(CD), units='unitless')
+
+        prob.add_pre_mission_systems()
+        prob.add_phases()
+        prob.add_post_mission_systems()
+
+        prob.link_phases()
+
+        prob.setup()
+
+        prob.set_initial_guesses()
+
+        prob.run_model()
+
+        CL_pass = prob.get_val("traj.cruise.rhs_all.core_aerodynamics.tabular_aero.CL")
+        CD_pass = prob.get_val("traj.cruise.rhs_all.core_aerodynamics.tabular_aero.CD")
+
+        assert_near_equal(CL_pass, CL_base, 1e-6)
+        assert_near_equal(CD_pass, CD_base, 1e-6)
+
+    def test_solved_aero_pass_polar_unique_abscissa(self):
```

Comment:
Was this test deleted in a regression, or was it moved from elsewhere? I remember writing this, but it appears as new in this diff.

Reply:
Great catch, it was unnecessarily duplicated in this PR, I fixed that now